### PR TITLE
Add a Heroku scenario to the README

### LINKED ISSUE #4: example scenarios in the readme
* [x] heroku
* [x] circleci
* [x] rails
* [x] rake

### PR DESCRIPTION
**Pull request by j0ni**
Closes #4

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -124,6 +124,21 @@ base_uri: "https://appcanary.com/api/v3"
...
+For Heroku rails deployments, everything should really "just work".
+
+1. Include the `appcanary` gem in your `Gemfile`.
+2. Ensure you have the `api_key` set to `ENV["APPCANARY_API_KEY"]`.
+3. Set the env var like this: `heroku config:set APPCANARY_API_KEY=xxxx -a yorapp`
+4. Once deployed, try `heroku run rake appcanary:check` to verify your dependencies.
+5. Optionally, set up a regular monitor update using the heroku scheduler:
+  1. `heroku addons:create scheduler:standard`
+  2. `heroku addons:open scheduler`
+  3. Add a job that executes `rake appcanary:update_monitor`
```
Comment (mveytsman) [inline README.md]: Is this a command I type or a thing I click on?

File: README.md [hunk 2]
```diff
@@ -124,6 +124,21 @@ base_uri: "https://appcanary.com/api/v3"
 monitor_name: "my_monitor"
 ```
 
+### Heroku
+
+For Heroku rails deployments, everything should really "just work".
+
+1. Include the `appcanary` gem in your `Gemfile`.
+2. Ensure you have the `api_key` set to `ENV["APPCANARY_API_KEY"]`.
```
Comment (mveytsman) [inline README.md]: how do I do this?

File: README.md (hunk 1 above)
Comment (j0ni) [reply README.md]: When you type the command in 2, it opens a browser on the scheduler. It's pretty obvious how to add a job once you do that.

Screenshots and stuff?

File: README.md (hunk 2 above)
Comment (j0ni) [reply README.md]: By following the instructions above this section, outlining all the ways you can configure the gem. Should I re-iterate all that here?

**Comment by j0ni** [comment]
Merging - @mveytsman let me know if you want to revisit.

**Pull request merged**